static/src/javascripts/lib/fetch-json.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import config from 'lib/config';
+import fetch from 'lib/fetch';
+
+function json(input: string, init: any = {}) {
```

Comment:
The object passed into this function is used to create another object that gets passed to `reqwest`. As such it doesn't have the same structure as `RequestInit` or `RequestOptions` but is something entirely bespoke for our application. 

I'll probably go with `Object` for now, since whatever type we define here will not be reusable.